.github/workflows/checks.yaml

```diff
@@ -246,7 +246,7 @@ jobs:
       - name: Install expect
```

Comment:
```suggestion
      - name: Install expect & bash-completion
```

Reply:
let's use the general world - dependencies...